framework/files/usr/lib/systemd/system/elemental-system-agent.service

```diff
@@ -15,3 +15,4 @@ StandardError=journal+console
 Environment="CATTLE_AGENT_CONFIG=/etc/rancher/elemental/agent/config.yaml"
 Environment="CATTLE_LOGLEVEL=debug"
 ExecStart=/usr/sbin/elemental-system-agent sentinel
+ExecStop=sleep 10
```

Comment:
This wont stop the agent from runnig, will it? This just substitutes the command that systemd will call to stop the service IIRC

Reply:
disregard this, manual says otherwise.